src/test/java/com/contrast/labs/ai/mcp/contrast/ADRServiceTest.java

```diff
@@ -575,4 +702,51 @@ private List<Attack> createMockAttacks(int count) {
 
         return attacks;
     }
+
+    /**
+     * Creates mock ProtectData for testing
+     */
+    private com.contrast.labs.ai.mcp.contrast.sdkexstension.data.ProtectData createMockProtectData(int ruleCount) {
```

Comment:
could ProtectData be imported? or does its name conflict with something else so it has to be fully qualified?